openedx/features/course_duration_limits/models.py

```diff
@@ -80,7 +80,7 @@ def has_full_access_role_in_masquerade(cls, user, course_key, course_masquerade)
         return False
 
     @classmethod
-    def enabled_for_enrollment(cls, enrollment=None, user=None, course_key=None):
```

Comment:
Are there no existing cases in the code where we call this without an enrollment?

Reply:
same comment as above